scripts/process/find_range.R

```diff
@@ -0,0 +1,18 @@
+#' Extract the min and max to use to scale circles in D3
+#' 
+process.find_range <- function(viz) {
+  
+  deps <- readDepends(viz)
+  wu_data_15_simple <- deps[["wu_data_15_simple"]]
+  col_names <- deps[["col_names"]][["new_names"]]
+  
+  water_use_df <- dplyr::select(wu_data_15_simple, col_names)
```

Comment:
wooaaahh, hold up! you can pass a character vector in to `dplyr::select`? i guess you can!
```r
> df <- data.frame(x=1,y=2,z=3)
> dplyr::select(df, c('x','y'))
  x y
1 1 2
```
when did that happen?

Reply:
I don't know, I was surprised myself!

Reply:
well, it's cool! i will definitely be using this - thanks for teaching me things today